adm_program/installation/db_scripts/update_3_3.xml

```diff
@@ -4,6 +4,7 @@
     <step id="20">UPDATE %PREFIX%_user_fields SET usf_name = 'SYS_STREET' WHERE usf_name_intern = 'STREET' AND usf_name = 'SYS_ADDRESS'</step>
     <step id="30">UPDATE %PREFIX%_user_fields SET usf_name = 'SYS_STREET' WHERE usf_name_intern = 'STREET' AND usf_name = 'Adresse'</step>
     <step id="40">UPDATE %PREFIX%_user_fields SET usf_name = 'SYS_STREET' WHERE usf_name_intern = 'STREET' AND usf_name = 'Address'</step>
+    <step id="45">UPDATE %PREFIX%_preferences SET prf_name = 'dates_default_list_configuration' WHERE prf_name = 'dates_default_list_configurati'</step>
```

Comment:
@Fasse typo?
`'dates_default_list_configurati'`

Reply:
no, this is a fix for an update from old versions, where prf_name length is only 30 chars